dashboard/test/controllers/api/v1/pd/workshop_enrollments_controller_test.rb

```diff
@@ -394,6 +399,23 @@ class Api::V1::Pd::WorkshopEnrollmentsControllerTest < ::ActionController::TestC
     assert_equal 2, destination_workshop.enrollments.length
   end
 
+  test 'edit' do
+    workshop = create :summer_workshop
+    enrollment = create :pd_enrollment, first_name: 'Rubeus', last_name: 'Hagrid', workshop: workshop
+
+    admin = create :workshop_admin
+    sign_in admin
+
+    post :edit, params: {
+      id: enrollment.id,
+      first_name: 'Harry',
+      last_name: 'Potter'
+    }
+
+    enrollment.reload
+    assert_equal({'first_name' => 'Harry', 'last_name' => 'Potter'}, enrollment.attributes.slice('first_name', 'last_name'))
```

Comment:
This seems ugly...a cleaner way to do this?

Reply:
compare first name and last name in separate asserts?